src/event-helper.js

```diff
@@ -50,11 +50,12 @@ export function createCustomEvent(win, type, detail, opt_eventInit) {
  * @param {string} eventType
  * @param {function(!Event)} listener
  * @param {boolean=} opt_capture
```

Comment:
Since it's our implementation and we already implement `once` through `listenOncePromise`, I thought there's really no point from passing an object when the only two options are `capture` and `passive`, but if you think it should be that way then sure!